backend/src/test/java/com/woowacourse/momo/controller/schedule/ScheduleControllerTest.java

```diff
@@ -70,4 +77,29 @@ void create() {
                 .then().log().all()
                 .statusCode(HttpStatus.OK.value());
     }
+
+    @DisplayName("약속 uuid와 참가자 이름으로 스케줄 조회를 요쳥하면 200OK를 반환한다.")
+    @Test
+    void findEach() {
+        Meeting meeting = meetingRepository.save(MeetingFixture.GAME.create());
+        attendeeRepository.save(AttendeeFixture.HOST_JAZZ.create(meeting));
+        Attendee attendee = attendeeRepository.save(AttendeeFixture.GUEST_DAON.create(meeting));
+        LocalDate now = LocalDate.now();
+        AvailableDate today = availableDateRepository.save(new AvailableDate(null, now, meeting));
+        AvailableDate tomorrow = availableDateRepository.save(new AvailableDate(null, now.plusDays(1), meeting));
+        availableDateRepository.save(new AvailableDate(null, now.plusDays(2), meeting));
+        List<Schedule> schedules = new ArrayList<>();
+        schedules.add(new Schedule(attendee, today, Timeslot.TIME_0300));
+        schedules.add(new Schedule(attendee, today, Timeslot.TIME_0400));
+        schedules.add(new Schedule(attendee, today, Timeslot.TIME_0500));
+        schedules.add(new Schedule(attendee, tomorrow, Timeslot.TIME_1600));
+        schedules.add(new Schedule(attendee, tomorrow, Timeslot.TIME_1700));
+        schedules.add(new Schedule(attendee, tomorrow, Timeslot.TIME_1300));
+        scheduleRepository.saveAll(schedules);
+
+        RestAssured.given().log().all()
+                .when().get("/api/v1/meeting/{uuid}/schedule?attendeeName=" + attendee.name(), meeting.getUuid())
+                .then().log().all()
+                .statusCode(HttpStatus.OK.value());
```

Comment:
```suggestion

        RestAssured.given().log().all()
                .when().given().queryParam(attendee.name(), meeting.getUuid())
                .get("/api/v1/meeting/{uuid}/schedule")
                .then().log().all()
                .statusCode(HttpStatus.OK.value());
```
이런 식으로 param을 설정할 수도 있어요. 제안만 해봅니다! ([참고](https://github.com/rest-assured/rest-assured/wiki/Usage#parameters))

Reply:
오! 이런 메서드가 있었네요.
적용해보도록 할게요~